### ISSUE TITLE
full mesh topology

### ISSUE THREAD

**Issue by prathibhacdac** [InnovateAsterisk/Browser-Phone#435]
Can we not have full mesh topology for peer to peer(1:1) video call in browser phone instead of using SFU?

**Comment by InnovateAsterisk** [comment]
No, Browser Phone was designed for Asterisk Conference (Conf_Bridge) in SFU and single stream (follow talker or marked).
